engine/storage/src/main/java/org/apache/cloudstack/storage/allocator/AbstractStoragePoolAllocator.java

```diff
@@ -55,41 +54,30 @@
 
 public abstract class AbstractStoragePoolAllocator extends AdapterBase implements StoragePoolAllocator {
     private static final Logger s_logger = Logger.getLogger(AbstractStoragePoolAllocator.class);
-    @Inject
-    StorageManager storageMgr;
-    protected @Inject
-    PrimaryDataStoreDao _storagePoolDao;
-    @Inject
-    VolumeDao _volumeDao;
-    @Inject
-    ConfigurationDao _configDao;
-    @Inject
-    ClusterDao _clusterDao;
-    protected @Inject
-    DataStoreManager dataStoreMgr;
-    protected BigDecimal _storageOverprovisioningFactor = new BigDecimal(1);
-    long _extraBytesPerVolume = 0;
-    Random _rand;
-    boolean _dontMatter;
-    protected String _allocationAlgorithm = "random";
-    @Inject
-    DiskOfferingDao _diskOfferingDao;
-    @Inject
-    CapacityDao _capacityDao;
+
+    protected BigDecimal storageOverprovisioningFactor = new BigDecimal(1);
+    protected String allocationAlgorithm = "random";
+    protected @Inject DataStoreManager dataStoreMgr;
+    @Inject PrimaryDataStoreDao storagePoolDao;
+    @Inject VolumeDao volumeDao;
+    @Inject ConfigurationDao configDao;
+    long extraBytesPerVolume = 0;
+    @Inject private CapacityDao capacityDao;
+    @Inject private ClusterDao clusterDao;
+    @Inject private StorageManager storageMgr;
+    @Inject private StorageUtil storageUtil;
```

Comment:
Seems that some variables can be protected, such as the `extraBytesPerVolume`, `storagePoolDao`, and others. The `storageMgr` can be private.

Reply:
Done